influxql/ast.go

```diff
@@ -857,6 +859,48 @@ func (s *SelectStatement) RewriteDistinct() {
 	}
 }
 
+// ColumnNames will walk all fields and functions and return the appropriate field names for the select statement
+// while maintaining sort order of the field names
```

Comment:
OK, minor but I would removed the word "sort" from the comment. All the code is doing is keeping the existing order.